src/dirtyfields/compat.py

```diff
@@ -13,6 +14,11 @@ def is_db_expression(value):
         return isinstance(value, (BaseExpression, Combinable))
 
 
+def is_deferred(model, field):
```

Comment:
I like the idea of using module names that are a bit more specialised than `utils.py`, where lot of different notions are mixed at the end. 

This one is related to field and db, maybe `db.py` (`fields.py` is usually more used for new fields creation) ? Open to better ideas :-)